### PR TITLE
add zsh-dropbox, zsh-bd, and solarized-man plugins

### PR DESCRIPTION
**Pull request by dreamfliper**
Is this the right way to make plugin request ?

### CODE REVIEW

**Comment by molovo** [comment]
It is absolutely, thanks for the PR. I'll review this later this evening and get it merged.

File: packages/zsh-dropbox [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "zsh-dropbox",
+  "description": "This plugin provides 2 apps: dropbox-cli and dropbox-uploader and completions for them.",
+  "repository": "https://github.com/zpm-zsh/zsh-dropbox",
+  "author": "Grigorii Horos"
+}
```
Comment (molovo) [inline packages/zsh-dropbox]: You haven't added a `bin`, `share` or `init` key here, so this package doesn't actually install anything. The two apps should go in `bin`, and the completion for them in `share`.

**Review by molovo — CHANGES_REQUESTED** [review]
I'm happy with the solarized-man and zsh-bd packages, they work great, thanks.

The zsh-dropbox package doesn't actually install anything (see https://github.com/zulu-zsh/index/pull/14/files#diff-a82bcee8f11a1f35557243e2fd93daebR6). If you could update that I'd be happy to merge.

Also, could you please rebase on master one last time? I've just had to push a fix to the CI procedure because it broke with the new version of ZUnit and the tests failed because of that. Thanks

**Comment by dreamfliper** [comment]
It seems dropbox-cli only works on Linux, because the official Dropbox client only provides command-line-interface on Linux platform.

**Comment by molovo** [comment]
@dreamfliper I'm not too worried about that. I expect there's other packages where that will be the case as well. I'll merge it now, and see if I can find a way of specifying OS requirements for packages in the future.

Thanks for the PR!

**Review by molovo — APPROVED** [review]

**Pull request merged**